deploy/olm/storageos/storageos.clusterserviceversion.yaml

```diff
@@ -318,14 +321,90 @@ spec:
       version: v1
       kind: StorageOSCluster
       displayName: StorageOS Cluster
-      description: StorageOS Cluster installs StorageOS in the cluster. It contains all the configuration for setting up a StorageOS cluster and also shows the status of the running StorageOS cluster.
+      description: StorageOS Cluster installs StorageOS in the cluster. It
+        contains all the configuration for setting up a StorageOS cluster and
+        also shows the status of the running StorageOS cluster.
       specDescriptors:
+      - description: Defines the various container images used in the cluster.
+        displayName: Images
+        path: images        
+      - description: The namespace to install the StorageOS cluster into.
+          `kube-system` is recommended so that StorageOS does not get evicted if
+          a node becomes over-allocated.
```

Comment:
Should we mention what's the default value?

Reply:
yes, thought the default is still `storageos`.  Should I change to `kube-system`?

Reply:
I'm not sure if just moving to `kube-system` will be enough. We need to set pod priority class to ensure we don't get evicted under resource pressure.

Reply:
I think for now, we should keep the wording, then in a separate PR we change the default and add the priority class?

Reply:
Added to the example instead of changing the default, which would be a breaking change.